packages/fluentui/react-northstar/src/themes/teams/colors.ts

```diff
@@ -432,7 +432,8 @@ export const colorScheme: ColorSchemeMapping<ColorScheme, TeamsColorNames> = {
     foreground2: colors.green[400],
     foreground3: colors.white,
     background: colors.green[300],
-    background1: colors.green[600],
+    background1: colors.green[50],
```

Comment:
Does changing this existing color scheme token affect any other component styles ?